doc/whats_new/v0.20.rst

```diff
@@ -77,6 +77,11 @@ Model evaluation
 - Added :class:`multioutput.RegressorChain` for multi-target
   regression. :issue:`9257` by :user:`Kumar Ashutosh <thechargedneutron>`.
 
+- Added the :class:`preprocessing.TransformedTargetRegressor` which transforms
```

Comment:
For some reason this had disappeared from what's new and I've just reinserted it :\